statsmodels/graphics/functional.py

```diff
@@ -638,24 +644,44 @@ def fboxplot(data, xdata=None, labels=None, depth=None, method='MBD',
     # Plot envelope of all non-outlying data
     lower_nonout = data[ix_nonout, :].min(axis=0)
     upper_nonout = data[ix_nonout, :].max(axis=0)
+    ax.plot(xdata, lower_nonout,
+            color=plot_opts.get('margin_line_color', 'blue'))
+    ax.plot(xdata, upper_nonout,
+            color=plot_opts.get('margin_line_color', 'blue'))
     ax.fill_between(xdata, lower_nonout, upper_nonout,
                     color=plot_opts.get('c_outer', (0.75, 0.75, 0.75)))
 
     # Plot central 50% region
+    ax.plot(xdata, lower,
+            color=plot_opts.get('margin_line_color', 'blue'))
+    ax.plot(xdata, upper,
+            color=plot_opts.get('margin_line_color', 'blue'))
     ax.fill_between(xdata, lower, upper,
                     color=plot_opts.get('c_inner', (0.5, 0.5, 0.5)))
 
+    # plot the vertical bar at middle points over x
+    bar_mid_val = (xdata[0] + xdata[-1]) / 2
```

Comment:
This looks like it is changing the existing plots and cannot be disabled.  Is this the case?

Reply:
Yeah, it is exactly the same as the figure in the paper.